exercise_2/src/exercise_2.cpp

```diff
@@ -0,0 +1,61 @@
+#include <stdio.h>
+#include <limits.h>
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+
+int const VERTICES {8};
+int const ROOT {0};
+
+int minDistance(int distance[], bool isVisited[])
+{
+   int min = INT_MAX, min_index;
```

Comment:
Initialize value. In case there was no vertices you would return uninitialized value.

Reply:
Done